js/renovation/scheduler/workspaces/base/time_panel/layout.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay,
+} from 'devextreme-generator/component_declaration/common';
+import { Row } from '../row';
+import { TimePanelCell as Cell } from './cell';
+import { ViewCellData } from '../../types';
+
+export const viewFunction = (viewModel: TimePanelTableLayout) => (
+  <table
+    className={`${viewModel.props.className} dx-scheduler-time-panel`}
```

Comment:
And this should be tested as well.
And seems that className should be the last of classes to override all the necessary css properties